lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -844,6 +908,23 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
     return std::make_pair(inputShape, outputShape);
   }
 
+  // Gets the ratio between the unknown dimensions in the input shape and the
+  // output shape. This ratio is used to match parallel unknown dimensions.
+  static std::pair<int64_t, int64_t>
+  getMultiplier(SmallVector<int64_t> inputShape,
+                SmallVector<int64_t> outputShape) {
+    int64_t inputMultiplier = 1;
+    int64_t outputMultiplier = 1;
```

Comment:
nit: no need to set these to 1. I would move the initialization down to where the final value is computed

Reply:
done